pom.xml

```diff
@@ -2332,6 +2337,21 @@
             </properties>
         </profile>
 
+        <profile>
+            <id>fast</id>
```

Comment:
We can update the use of `-Pfast` profile in these places.
`AllKyuubiConfiguration.scala` , `TestUtils.scala`, `developer.md`

Reply:
Good suggestion. However, `fast` profile is skipping tests run that prevent running actual tests and ignoring `-DwildcardSuites`. As result, we will have to use `fast` profile while explicitly set `skipTest` to false with ` -Pfast -DskipTests=false`.

Reply:
`KYUUBI_UPDATE=1 build/mvn clean install -Pfast -DskipTests=false -DwildcardSuites=org.apache.kyuubi.config.AllKyuubiConfiguration`
It's not generating new settings.md as expected.